packages/react-components/react-rating-preview/src/components/RatingDisplay/useRatingDisplay.tsx

```diff
@@ -61,7 +61,11 @@ export const useRatingDisplay_unstable = (
     }),
     countText: slot.optional(props.countText, {
       renderByDefault: count !== undefined,
-      defaultProps: { children: count?.toLocaleString(), id: countTextId, 'aria-hidden': true },
+      defaultProps: {
+        children: count && count >= 0 && Math.round(count).toLocaleString(),
```

Comment:
I don't think we should make this change. It's true that it doesn't make sense for `count` to be a non-integer. However, it's a bug in the code that _uses_ RatingDisplay if they provide a non-integer value; it's not a bug in RatingDisplay itself for displaying the value it's given. I don't think we should "magically" fix it and hide the bug in the calling code.

That said, I might be misunderstanding why you're making this change -- please let me know if there's a reason I'm missing. Thanks!

Reply:
Makes sense to add documentation that `count` should always be a positive integer and not make a "magic" fix